content/en/blog/_posts/2025-04-23-userns-enabled-by-default/index.md

```diff
@@ -0,0 +1,282 @@
+---
+layout: blog
+title: "Kubernetes 1.33: User Namespaces enabled by default!"
+date: 2025-04-23
+draft: true
+slug: userns-enabled-by-default
+author: >
+  Rodrigo Campos Catelin (Microsoft),
+  Giuseppe Scrivano (Red Hat),
+  Sascha Grunert (Red Hat)
+---
+
+In Kubernetes 1.33 support for user namespaces is enabled by default. This means
+that, when the stack requirements are met, pods can opt-in to use user
```

Comment:
I'm pinning the comment here, but probably there is a better place to address it. After reading the blog (and I'm new to node stuff so I may not be the best representative of a reader here) I'm still puzzled how to use it / opt-in (Is this opted in by the pod spec?). I guess this is under the "Demos" section in the links, but maybe embedding something like a code/yaml snippet showing how to use it would help.

Reply:
Awesome, thank you for the example! I didn't expect this to be this easy indeed. 

I also like the note below the example, IIUC, it explains  why this is not by default, because it would be breaking to users who need the root privileges on the host actually. I guess, if this feature was supported on day one, then `hostUsers: false` would be default behavior, and `hostUsers: true` would be opt-in (to get the root privileges)? is my understanding of the context more or less correct? 

And +1 for adding the example of a legitimate use-case of not using the feature "for example an app that needs to load a kernel module".

Reply:
Yeap, exactly, you got everything right! :)